### ISSUE TITLE
breaking update?

### ISSUE THREAD

**Issue by okwme** [MetaMask/metamask-extension#2477]
after unlocking the wallet i got a grey screen.
i restarted the extension, then restarted my computer, both ended in the same behavior.
i uninstalled and reinstalled metamask, used my seed to unlock and always the same.

using Version 61.0.3163.100 (Official Build) (64-bit)
MacBook Air running Sierra 10.12.5

i inspected the popup and this was in my error log:

````
Warning: Failed prop type: Invalid prop `selected` of type `number` supplied to `AccountDropdowns`, expected `string`.
    in AccountDropdowns (created by Connect(AccountDropdowns))
    in Connect(AccountDropdowns) (created by App)
    in div (created by App)
    in div (created by App)
    in div (created by App)
    in div (created by App)
    in App (created by Connect(App))
    in Connect(App) (created by Root)
    in Provider (created by Root)
    in Root
printWarning @ popup.js:38203
popup.html:1 Error in event handler for (unknown): TypeError: Cannot read property 'toLowerCase' of null
    at Object.exports.toChecksumAddress (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:29477:44)
    at module.exports (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:93040:29)
    at PendingTx.miniAccountPanelForRecipient (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:88289:8)
    at PendingTx.render (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:88139:33)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:55367:21
    at measureLifeCyclePerf (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:54647:12)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:55366:25)
    at ReactCompositeComponentWrapper._renderValidatedComponent (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:55393:32)
    at ReactCompositeComponentWrapper.performInitialMount (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:54933:30)
    at ReactCompositeComponentWrapper.mountComponent (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/scripts/popup.js:54829:21)
popup.js:56726 Uncaught TypeError: Cannot read property '__reactInternalInstance$nllbge6hcb' of null
    at Object.getClosestInstanceFromNode (popup.js:56726)
    at findParent (popup.js:59174)
    at handleTopLevelImpl (popup.js:59203)
    at ReactDefaultBatchingStrategyTransaction.perform (popup.js:63704)
    at Object.batchedUpdates (popup.js:58894)
    at Object.batchedUpdates (popup.js:61864)
    at dispatchEvent (popup.js:59283)
````

**Comment by Zanibas** [comment]
Could you provide us what your current MetaMask version is, via chrome://extensions?

**Comment by oceanapplications** [comment]
I'm also receiving this error on 3.12.0.

**Comment by oceanapplications** [comment]
For me this error is caused by initiating a transaction with an invalid address.   Every invalid address transaction causes the same breaking error.  The only fix I could find is to reinstall the extension. 

This seems like it could be used maliciously to lock people out of their metamask.  It can be executed without any action from the user.

**Comment by jjjrmy** [comment]
Same issue on current

**Comment by tmashuang** [comment]
What format is the address in? I believe there should be an error when an invalid address is submitted as for the error message.

**Comment by whydna** [comment]
Running into the same problem. Anyone here have a workaround?

**Comment by tmashuang** [comment]
This should have been fixed in #3146. Please comment if this is still an issue.
